stdlib/public/core/FixedPoint.swift.gyb

```diff
@@ -244,10 +244,10 @@ public struct ${Self}
   /// byte order if necessary.
   @_transparent public
   init(bigEndian value: ${Self}) {
-#if arch(i386) || arch(x86_64) || arch(arm) || arch(arm64)
+#if arch(i386) || arch(x86_64) || arch(arm) || arch(arm64) || arch(powerpc64le)
     self = ${Self}(Builtin.int_bswap_${BuiltinName}(value._value) )
 #else
-    _UnsupportedArchitectureError()
```

Comment:
Please keep the unsupported error so that we get a loud error for new architectures.  Just add an explicit `#elseif arch(powerpc64)` for BE (here and below).